src/D2L.Security.OAuth2/D2L.Security.OAuth2.csproj

```diff
@@ -2,7 +2,7 @@
 	<Sdk Name="Microsoft.Build.CentralPackageVersions" />
 
 	<PropertyGroup>
-		<TargetFrameworks>net472;net48</TargetFrameworks>
+		<TargetFrameworks>net472;net50</TargetFrameworks>

```

Comment:
Doing .NET 5.0 now and using System.Text.Json from the framework.

Doing netstandard20 instead would be easy, but then we might want to consider dropping the net472 target entirely.

There are two issues:

* The LMS needs net472 for the System.Web support. The library APIs would have to be changed to avoid this.
* We're punting on using System.Text.Json inside the LMS because we aren't sure about performance (and maybe some assembly spaghetti problems).
  * We could measure this and determine that it's fine
  * Or we could finish upgrading Newtonsoft in the LMS and stick to newtonsoft in here, possibly just for a netstandard build.

We can always change our minds on this easily later.